portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,77 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+ *{
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+    } 
+    
+#resume-header {
+    margin-top: 20px;
+    margin-bottom: 20px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#profile-picture {
+    width: 100%;
 }
+
+#resume-footer {
+    padding: 20px;
+    text-align: center;
+}
+
+.main-nav
+{ /* Para quitarle los estilos a las listas*/
+    background-color: darkgray;
+}
+
+.main-nav__item
+{
+ display: inline-block;
+}
+
+.main-nav__item a 
+{
+    color:black;
```

Comment:
For consistency, leave a space between the property and the value. In this case, add a space between `color:` and `black`.